collectors/aws/elasticbeanstalk/describeConfigurationSettings.js

```diff
@@ -0,0 +1,24 @@
+var AWS = require('aws-sdk');
+var async = require('async');
+
+module.exports = function(AWSConfig, collection, callback) {
+    var elasticbeanstalk = new AWS.ElasticBeanstalk(AWSConfig);
+
+    async.eachLimit(collection.elasticbeanstalk.describeEnvironments[AWSConfig.region].data, 15, function(environment, cb) {
+        var params = {
+            ApplicationName: environment.ApplicationName,
+            EnvironmentName: environment.EnvironmentName
+        };
+        elasticbeanstalk.describeConfigurationSettings(params, function(err, data) {
+            collection.elasticbeanstalk.describeConfigurationSettings[AWSConfig.region][environment.EnvironmentName] = {};
```

Comment:
@giorod3 @AkhtarAmir - I am concerned about using the environment name here as the unique key. Different projects can share the same environment name. For example, I can have the following:
```
Project: xyz
Environment: prod

Project: abc
Environment: prod
```

In this case, the second will overwrite the first. I think we need to either use a full ARN or some combination of app name + environment name,

Reply:
Yes ARN makes more sense. Fixed